### PR TITLE
RELATED: ONE-4163 Add locale to DateFilter component, remove unused e…

### PR DESCRIPTION
**Pull request by petrjanu**
…xample callbacks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/159/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/40/ : SUCCESS in 7s

File: website/i18n/en.json [hunk 1]
```diff
@@ -4,1088 +4,147 @@
...
-      "Support": "Support",
-      "Introduction": "Introduction",
-      "Tutorials": "Tutorials",
-      "React Components": "React Components",
-      "Reference": "Reference",
-      "Tools": "Tools",
-      "License": "License",
-      "FAQ": "FAQ"
-    }
+    "about_gooddataui": "About GoodData.UI",
```
Comment (vlastikcz) [inline website/i18n/en.json]: what is this?

Comment (petrjanu) [reply website/i18n/en.json]: I don't know. It was formatted in one of my previous PR and this reverts the change back. This reformat happened after starting up docusaurus dev server.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/160/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/44/ : SUCCESS in 18s

**Pull request merged**